docs/modules/automation-testing/pages/start-a-mixed-session.adoc

```diff
@@ -13,20 +19,58 @@ image:automation-testing:launch-device-context.png[width=800,alt="A context of l
+
+=== If the first session is automation
+
+Make sure you are logged in as the same user that started the automation session.
+
+Follow one of the following possible methods to start a manual session in an ongoing automation session:
+
+* From the *Device list*, select the *Launch* button while the device was running automation to enter the mixed session:
+
++
+
+image:automation-testing:launch-mixed-session-device-list.png[width=1000,alt="A utilizing device with a Launch button to start a mixed session"]
+
+* Open the *Session Overview* page of the session, then select the *Session Viewer* button:
+
++
+
+image:automation-testing:launched-mixed-session-session-overview.png[width=1000,alt="The Session Viewer button in Session Overview of an ongoing automation session"]
+
+* In the session list, highlight the ongoing automation session (by clicking the session name) then choose *Session Viewer* on the action menu:
+
++
+
+image:automation-testing:launched-mixed-session-session-list-hightlight.png[width=1000,alt="The Session Viewer option when highlighting an ongoing automation session in session list"]
+
+* Alternatively, in the session list, right-click the session, and select *Session Viewer*:
+
++
+
+image:automation-testing:launch-mixed-session-session-list-right-click.png[width=1000,alt="The Session Viewer option when right-clicking an ongoing automation session in session list"]
 
 Now you can view your script running on the device and fix issues in realtime.
 
 == Limitations/Notes
 
-* In a Mixed session, both the Manual and Automation session must be initiated by the same user.
+* In a mixed session, both the Manual and Automation session must be initiated by the same user.
 
 * Mixed session are only available for Kobiton 4+ devices.
 
-* Mixed sessions can only be initiated with Appium automation tests; other automation frameworks, e.g. Espresso, XCUITest, are not currently supported.
+* Mixed sessions can only be initiated with Appium or Basic Appium 2 automation tests; other automation frameworks, e.g. Espresso, XCUITest, are not currently supported.
+
+* Manual actions in a mixed session is not yet available when a Basic Appium 2 script is running.
 
 * Session Explorer is not available for Mixed Sessions; i.e. it's not possible to view detailed steps and screenshots for Mixed Sessions.
 
 * Mixed sessions cannot be turned into test cases. They cannot be used for our AI functionality: Scriptless, generating Appium Scripts, self-healing.
 
-* When you have a Mixed Session on a Public Device, you will be charged minutes for the total time usage on the device; you will not be double-charged for an automation and a manual session.
+* When you have a mixed Session on a Public Device, you will be charged minutes for the total time usage on the device; you will not be double-charged for an automation and a manual session.
+
+* A mixed session counts as 1 parallel test session, although it includes a manual and an automation session.
```

Comment:
A mixed session counts as 1 parallel test session, even though it includes both a manual and an automation session.